src/core/layout/qgslayoutitempolyline.h

```diff
@@ -185,6 +188,21 @@ class CORE_EXPORT QgsLayoutItemPolyline: public QgsLayoutNodesItem
      */
     double arrowHeadStrokeWidth() const { return mArrowHeadStrokeWidth; }
 
+    /**
+     * Returns a path representing the outline of the stroked polyline.
+     */
+    QPainterPath shape() const override; /*{
```

Comment:
Can you drop the commented code from this?

Reply:
Done